ddtrace/debugging/_encoding.py

```diff
@@ -386,6 +445,40 @@ def add_tags(payload):
         payload["ddtags"] = config.tags
 
 
+def format_captured_value(value):
+    # type: (Any) -> str
+    v = value.get("value")
+    if v is not None:
+        return v
+
+    es = value.get("elements")
+    if es is not None:
+        return "%s(%s)" % (value["type"], ", ".join(format_captured_value(e) for e in es))
+
+    es = value.get("entries")
+    if es is not None:
+        return "{%s}" % ", ".join(format_captured_value(k) + ": " + format_captured_value(v) for k, v in es)
+
+    fs = value.get("fields")
+    if fs is not None:
+        return "%s(%s)" % (value["type"], ", ".join("%s=%s" % (k, format_captured_value(v)) for k, v in fs.items()))
+
+    return "%s()" % value["type"]
+
+
+def format_message(function, args, retval=None):
+    # type: (str, Dict[str, Any], Optional[Any]) -> str
+    message = "%s(%s)" % (
+        function,
+        ", ".join(("=".join((n, format_captured_value(a))) for n, a in args.items())),
+    )
+
+    if retval is not None:
+        return "\n".join((message, "=".join(("@return", format_captured_value(retval)))))
```

Comment:
In the Java tracer we also print the locals in the summary message - we should do that here as well 🙂 

Since we print the return value separately, we should exclude `@return` from the serialised locals - https://github.com/DataDog/dd-trace-java/pull/4011#discussion_r994473993

Reply:
Sure, we can do that, but I suppose this only applies to line probes, in which case we don't have a `@return`? We don't have access to locals for function probes in Python.

Reply:
Didn't think of that, but now I remember we're wrapping the function for method probes. In Java we have the locals so the comment about `@return` applies there.

IMO we should print the locals in the summary when we have them, but it's fine to do that change in another PR.